src/Phansible/Role.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Phansible;
+
+interface Role
```

Comment:
I would follow the naming conventions from the php-fig group: https://github.com/php-fig/fig-standards/blob/master/bylaws/002-psr-naming-conventions.md  and rename it to RoleInterface. Same for the new interface you created.